src/data-search.js

```diff
@@ -45,8 +45,7 @@ export async function getData () {
 
     // Find and save final link for each paper's DOI
     _.forIn(papers, async function (value, key) {
-      const doiLink = `https://doi.org/${value.doi}`;
-      const finalURL = await getFinalURL(doiLink);
+      const finalURL = `https://www.biorxiv.org/content/${value.doi}v${value.version}`;
```

Comment:
This isn't how you want to iterate over the elements of papers.

Reply:
I see, could you please elaborate on why this isn't ideal?

Reply:
In this case `forIn` works but in general won't always be the case. I think you want for...of.
You don't use the `key` in the function. The `async` keyword is unnecessary.

Reply:
That makes sense. My mistake, I had key in there just so I could see the key value while debugging but forgot to take it out after. I'll look into this, thanks!